### PR TITLE
Installer updates

### PR DESCRIPTION
**Pull request by deiteris**
Addressing https://github.com/SlimeVR/SlimeVR-Installer/issues/33 and https://github.com/SlimeVR/SlimeVR-Installer/issues/32.

### CODE REVIEW

**Review by Eirenliel — APPROVED** [review]

**Comment by Eirenliel** [comment]
Please resolve conflicts, thanks

File: windows/web/slimevr_web_installer.nsi [hunk 1]
```diff
@@ -48,47 +48,20 @@ Var OPEN_DOCUMENTATION
 # Detected Steam folder
 Var STEAMDIR
 
-# Detected or specified SteamVR folder
-Var STEAMVRDIR
-Var STEAMVRDIR_TEXT
-Var STEAMVRDIR_DEST
-
 # Init functions start #
 # Detect Steam installation and prevent installation if none found
```
Comment (ButterscotchV) [inline windows/web/slimevr_web_installer.nsi:52]: It doesn't do this anymore
```suggestion
```

**Review by ButterscotchV — CHANGES_REQUESTED** [review]

File: windows/web/slimevr_web_installer.nsi [hunk 2]
```diff
@@ -238,15 +209,15 @@ Function endPageLeave
...
+        CreateShortcut "$SMPROGRAMS\SlimeVR Server\SlimeVR Server.lnk" "$INSTDIR\slimevr.exe" "" "$INSTDIR\run.ico"
     ${Else}
         Delete "$SMPROGRAMS\Uninstall SlimeVR Server.lnk"
         Delete "$SMPROGRAMS\SlimeVR Server.lnk"
         RMdir /r "$SMPROGRAMS\SlimeVR Server"
     ${Endif}
 
     ${If} $1 = 1
-        CreateShortcut "$DESKTOP\SlimeVR Server.lnk" "$INSTDIR\slimevr-ui.exe" "" "$INSTDIR\run.ico"
+        CreateShortcut "$DESKTOP\SlimeVR Server.lnk" "$INSTDIR\slimevr.exe" "" "$INSTDIR\run.ico"
```
Comment (ButterscotchV) [inline windows/web/slimevr_web_installer.nsi:220]: You just removed `run.ico` later in this PR, setting the icon here doesn't seem like a good idea

File: windows/web/slimevr_web_installer.nsi [hunk 3]
```diff
@@ -356,24 +286,20 @@ Section "SlimeVR Server" SEC_SERVER
...
+    nsisunz::Unzip "$TEMP\SlimeVR-win64.zip" "$TEMP\SlimeVR\"
     Pop $0
     DetailPrint "Unzipping finished with $0."
 
     DetailPrint "Copying SlimeVR Server to installation folder..."
     CopyFiles /SILENT "$TEMP\SlimeVR\SlimeVR\*" $INSTDIR
 
-    # Include modified run.bat that will run bundled JRE
-    File "run.bat"
-    File "run.ico"
```
Comment (ButterscotchV) [inline windows/web/slimevr_web_installer.nsi:375]: `run.ico` is still used for the shortcut and the registry entry for the program, it should still be included unless you change or remove those usages, I believe the shortcut should work fine without an icon as long as the exe has an icon?

**Review by ButterscotchV — CHANGES_REQUESTED** [review]
Does not resolve issue #33.

`slimevr-ui.exe` should probably be removed or renamed during update in order to remove the old version (unless it already does that, I'm unsure).
> `slimevr-ui.exe` should be renamed to `slimevr.exe` (will be updated in the release after this chane released), start menu and desktop shortcut should point to that

`run.ico` is not renamed or removed on update and `run.bat` is not removed on update.
> `run.bat` & `run.ico` files should be removed during an update. If icon is needed, it can be used from `slimevr.exe` or it can be renamte to `icon.ico`

File: windows/web/slimevr_web_installer.nsi (hunk 3 above)
Comment (deiteris) [reply windows/web/slimevr_web_installer.nsi:378]: Fixed. Yes, exe should work.

File: windows/web/slimevr_web_installer.nsi (hunk 2 above)
Comment (deiteris) [reply windows/web/slimevr_web_installer.nsi]: Yeah, forgot to remove. Fixed.

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**